serial-prober.js

```diff
@@ -151,6 +151,10 @@ class SerialProber {
             break;  // probe succeeded.
           } catch (err) {
             // probe failed.
+            console.log('Port', port.comName,
```

Comment:
If you instead let this error bubble up to the caller (i.e. the zigbee adapter), it will be caught and sent to the UI via the `errorCallback`.